aircraft_heirs.py

```diff
@@ -0,0 +1,171 @@
+import random
+from abc import ABC
+
+from aircraft import Aircraft
+
+
+class WaterLandingMixin:
+    @staticmethod
+    def do_water_landing():
+        print('Performing water landing')
+
+
+class BombThrowingMixin:
+    @staticmethod
+    def throw_bombs():
+        print('Throwing bombs')
+
+
+class PassengerAircraft(Aircraft, ABC):
+    def __init__(self, aircraft_id, aircraft_type, width, landing_length, num_of_seats,
+                 destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates):
+        super().__init__(aircraft_id, aircraft_type, width, landing_length, num_of_seats,
+                         destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates)
+        self.is_passenger = True
+        self.passengers = []
+
+    def release_cabin(self, airport):
+        if hasattr(airport, 'passengers'):
+            airport.passengers.extend(self.passengers)
+            [setattr(passenger, 'place', 'airport') for passenger in airport.passengers]
+            self.passengers = []
+
+    def fill_cabin(self, airport):
+        num_passengers_to_fill = int(len(airport.passengers) * 0.15)
+        passengers_to_fill = airport.passengers[:num_passengers_to_fill]
+        [setattr(passenger, 'place', 'aircraft') for passenger in airport.passengers]
```

Comment:
you made all the passengers belong to aircraft here.
also the code could be refactored